### ISSUE TITLE
Disable sorting on table 

### ISSUE THREAD

**Issue by genogeno** [Budibase/budibase#9072]
**Describe the feature request**
- Currently by default you can sort the columns of the table.  Could you please add a property to have the ability to turn off any sorting at all. So that it won't call the backend again and again for sorting.  

**Screenshots**

**Comment by aptkingston** [comment]
Do you mean just basically disable the ability to click on the table column headers to change the sorting? We have this functionality but it just isn't exposed as a setting, so this would be easy to add.

**Comment by genogeno** [comment]
> Do you mean just basically disable the ability to click on the table column headers to change the sorting? We have this functionality but it just isn't exposed as a setting, so this would be easy to add.

Yes, exactly.  We want users not to click on the headers itself and do nothing with it.  

- User cannot click the table header
- No calls to API will be performed when user clicks on column header as disabled.
-

**Comment by genogeno** [comment]
@aptkingston thank you for the reply.  Any ballpark date that this feature can be enabled? thanks!

**Comment by Hertizch** [comment]
I would also like this setting. My rows should not be sorted like this.
